integration/testnet/util.go

```diff
@@ -105,3 +108,50 @@ func WriteJSON(path string, data interface{}) error {
 	err = ioutil.WriteFile(path, marshaled, 0644)
 	return err
 }
+
+// rootProtocolJsonWithoutAddresses strips out all node addresses from the root protocol json file specified as srcFile
+// and creates the dstFile with the modified contents
+func rootProtocolJsonWithoutAddresses(srcfile string, dstFile string) error {
```

Comment:
I think the part of this function which modifies the `Snapshot` should be exported, and not in a testing-specific package since I would imagine we will use this in the tool used to distribute the stripped snapshot file. 
Eg:
```go
// read root snapshot
strippedRootSnapshot := protocol.StrippedSnapshot(rootSnapshot) // the part we can re-use
WriteJSON(strippedRootSnapshot)
```

Reply:
Added a new function to `protocol/inmem/snapshot.go`